packages/fxa-auth-server/docs/swagger/descriptions.ts

```diff
@@ -0,0 +1,141 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
If you take me up on my other comment on renaming `account-docs` etc., you could potentially stick this file in a shared dir, like `docs/swagger/shared/descriptions.ts`. That way, the non-shared API docs can just live in `docs/swagger/account.ts` etc. and then the utils etc. can be in `/shared` but it's great either way 😍